packages/wallets/src/features/cashier/modules/DepositCrypto/components/DepositCryptoDisclaimers/DepositCryptoDisclaimers.tsx

```diff
@@ -42,7 +42,7 @@ const DepositCryptoDisclaimers = () => {
                 </div>
             </InlineMessage>
             <p className='wallets-deposit-crypto-disclaimers__note' style={{ fontSize: isMobile ? ' 1rem' : '1.2rem' }}>
```

Comment:
Can we put the styles in css? You can use `@include mobile` for that

Reply:
```suggestion
<div className=` your className`>
     <WalletText size='md' weight='bold'>Note:</WalletText>{" "}
     <WalletText size='md'>&nbsp;You’ll receive an email when your deposit start being processed.</WalletText>
```

Reply:
Updated